src/Orleans.TestingHost/Grains/ITestClusterManagementGrain.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Orleans.TestingHost.Grains
+{
+    public interface ITestClusterManagementGrain : IGrainWithGuidKey
+    {
+        Task<T> GetService<T>();
```

Comment:
I wouldn't add this grain here (in the package that we ship) since almost no service would be serializable. The only one I can think of is the legacy config objects, so wouldn't pollute the surface with this.

Reply:
Alright, should I make a more specific legacy-themed grain to get config objects or try to retrieve them some other way?

Reply:
If it was just to get the gateway endpoint, you probably don't even need that, you can derive it from the options, or even ask the client